frontend/apps/reader/modules/readerdevicestatus.lua

```diff
@@ -101,10 +110,10 @@ function ReaderDeviceStatus:addToMainMenu(menu_items)
         text = _("Device status alerts"),
         sub_item_table = {},
     }
-    if self.has_battery then
+    if Device:hasBattery() then
         table.insert(menu_items.device_status_alarm.sub_item_table,
             {
-                text = _("Low battery level"),
+                text = _("Low or high battery level"),
```

Comment:
Battery buddy? :-P

Reply:
Maybe just "Battery level"? Explanation is in the Thresholds dialog.

Reply:
Yes, just battery level sounds better.